src/templates/template.ts

```diff
@@ -1,27 +1,32 @@
 import pug from 'pug';
 import { join } from 'path';
+import juice = require('juice');
 import { Connection } from 'typeorm';
+import htmlToText = require('html-to-text');
 
 import { Message } from '../worker';
 import { getMigrationSummary } from '../query';
+import { Template } from './templateInterface';
 
 let template: string;
 
 export const loadTemplate = async (
   connection: Connection,
   message: Message
-): Promise<any> => {
+): Promise<Template> => {
   const migrationSummary = await getMigrationSummary(connection, message);
 
+  const { source, migrationFailed } = message;
+  template = source === 'failqueue' && !migrationFailed ? 'migration' : source;
```

Comment:
encapsulate this in a descriptive function

Reply:
worked on it, check it out.